app/helpers/users_helper.rb

```diff
@@ -0,0 +1,5 @@
+module UsersHelper
+  def current_user? user
```

Comment:
Sao phải define thêm cái helper này trong khi ở cái `SessionHelper` đã có và được include ở toàn app trong `ApplicationController` rồi ?

Reply:
dạ để em sửa lại